components/brave_new_tab_ui/components/default/braveToday/customize/FeedCard.tsx

```diff
@@ -69,8 +69,8 @@ export default function FeedCard (props: {
   const publisher = usePublisher(props.publisherId)
   const { followed, setFollowed } = usePublisherFollowed(props.publisherId)
 
-  const backgroundColor = publisher.backgroundColor || getCardColor(publisher.feedSource?.url || publisher.publisherId)
-  const { url: coverUrl, setElementRef } = useLazyUnpaddedImageUrl(publisher.coverUrl?.url, {
+  const backgroundColor = publisher?.backgroundColor || getCardColor(publisher?.feedSource?.url || publisher?.publisherId)
```

Comment:
Note: The fix doesn't actually need this but its should prevent future crashes in the same vein